### PR TITLE
Update DesignRationales.md

### PR DESCRIPTION
**Pull request by dtgerson**
adding more rationales and cleaning stuff up

### CODE REVIEW

File: DesignRationales.md [hunk 1]
```diff
@@ -96,33 +111,73 @@ More detailed thoughts on this [here](https://docs.google.com/document/d/1_2f-MA
 ## Filters
 * Filters on inlet/outlet to prevent contaminents from the surrounding environment getting into the device
 * Filter on patient circuit to avoid device contamination and spreading particles between patients
-* We're currently looking at HEPA filters since they are commonly avaible and cheap - but we're also evaluating other options. More info on HEPA filters: https://respiraworks.slack.com/archives/C0137BLF17A/p1589983907134500?thread_ts=1589940279.125800&cid=C0137BLF17A
+* We're currently looking at HEPA filters since they are commonly avaible and cheap - but we're also evaluating other options. [More info on HEPA filters](https://respiraworks.slack.com/archives/C0137BLF17A/p1589983907134500?thread_ts=1589940279.125800&cid=C0137BLF17A)
```
Comment (gracecanfield) [inline DesignRationales.md:114]: @dtgerson Can we pull this content into GitHub? This is supposed to be an open-source page available to anyone, but that slack link will only work for members of the team who have RW slack accounts.

File: DesignRationales.md [hunk 2]
```diff
@@ -96,33 +111,73 @@ More detailed thoughts on this [here](https://docs.google.com/document/d/1_2f-MA
...
+* **#todo** - need more here.
+* **#todo** - add diagrams from design reviews and such
 
 ## Software
-* #todo - should figure out some stuff for this
 
-## GUI
-* #todo - should figure out some stuff for this
+* **#todo** - add diagrams from design reviews and such
+* **Separation of UI Controller and Cycle Controller**
```
Comment (gracecanfield) [inline DesignRationales.md:151]: Should this whole section go under the "Controller" section subheading below?

File: DesignRationales.md (hunk 1 above)
Comment (dtgerson) [reply DesignRationales.md:116]: great point - made this a todo above where I added a note about the importance of open source - will take some more time to figure out what info to go move to git, but will do it

File: DesignRationales.md (hunk 2 above)
Comment (dtgerson) [reply DesignRationales.md]: tried to clean this up a bit

File: DesignRationales.md (hunk 1 above)
Comment (gracecanfield) [reply DesignRationales.md:116]: Okay, cool

File: DesignRationales.md (hunk 2 above)
Comment (gracecanfield) [reply DesignRationales.md]: Looks great!

**Pull request merged**